src/test/xattr_bench.cc

```diff
@@ -30,8 +30,6 @@
 #include <boost/random/binomial_distribution.hpp>
 #include <gtest/gtest.h>
 
-#include "include/unordered_map.h"
```

Comment:
needed here?

Reply:
Is it? I don't think so.

Reply:
Let's remove it.

Reply:
Remove what? My PR does remove this line, because it's unused by this source file.

Reply:
Sorry - my mistake!